6-pubsub/worker.js

```diff
@@ -175,8 +175,12 @@ function queryBooksApi (query, cb) {
 }
 // [END query]
 
-exports.app = app;
-exports.subscribe = subscribe;
-exports.processBook = processBook;
-exports.findBookInfo = findBookInfo;
-exports.queryBooksApi = queryBooksApi;
+app.mocks = {
```

Comment:
@jmdobry this was the quickest way to do things, but it feels kludgey to me.

Should we find a better way to expose _both_ the app and the mocks, even if it means modifying `nodejs-repo-tools` further?